ui/app/helpers/supported-secret-backends.js

```diff
@@ -11,6 +11,7 @@ const SUPPORTED_SECRET_BACKENDS = [
   'transit',
   'kmip',
   'transform',
+  'keymgmt',
```

Comment:
We don't want to commit this until phase 2

Reply:
Gotcha I removed it for now.